arch/risc-v/src/mpfs/mpfs_emmcsd.c

```diff
@@ -1339,6 +1343,12 @@ static void mpfs_sdcard_init(struct mpfs_dev_s *priv)
   putreg32(LIBERO_SETTING_IOMUX2_CR_SD, MPFS_SYSREG_IOMUX2);
   putreg32(LIBERO_SETTING_IOMUX6_CR_SD, MPFS_SYSREG_IOMUX6);
```

Comment:
Do you actually even need to write all of these IOMUX registers when using the GPIO?

Reply:
Tested. IOMUXing needed.